packages/TelegramClient-UI.package/TCUEmojiHelper.class/class/embedEmojisInto..st

```diff
@@ -0,0 +1,17 @@
+utility
+embedEmojisInto: aString
+
+	| reencoded runs |
+	
+	reencoded := (aString convertToEncoding: 'UTF-16') convertFromEncoding: 'UTF-16'.
+	runs := self createUnicodeRunsFrom: reencoded.
+	^ runs inject: Text new
+		into: [:aggregatedText :runOrChar |
+			(runOrChar species = OrderedCollection) ifTrue: [
```

Comment:
Not quite, the original string was already processed by `createUnicodeRunsFrom` here. So `runs` now is an OrderedCollection of Characters or OrderedCollections which themselves only contain Characters. e. g. runs could look something like this `a OrderedCollection {$A . $B . a OrderedCollection {$127643 $114592} . $C }`

Since we found and already used `isCollection`, lets use it here too.

Reply:
Okay great